src/components/App.js

```diff
@@ -0,0 +1,19 @@
+import '../App.css';
+import Display from './Display';
+// import Button from './Button';
+import ButtonPanel from './ButtonPanel';
```

Comment:
It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, its cleaner to use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))